### PR TITLE
[amo] Custom labels & Tests

### LINKED ISSUE #996: Mozilla Add-ons badge cannot override label

### LINKED ISSUE #1026: How to use Mozilla AMO badges?
How to use "Mozilla Add-on", such as:
version: [Mozilla Add-on](https://addons.mozilla.org/en-US/firefox/addon/facebook-post-screenshot/)
rating: [Mozilla Add-on](https://addons.mozilla.org/pl/firefox/addon/facebook-post-screenshot/)

SVG file probably should take add-on name (or something else) as an argument.
Currently all values are hard-coded and thus mentioned badges are useless.

### PR DESCRIPTION
**Pull request by RedSparr0w**
Closes #1026 
Closes #996 
Added tests

URL | Badge
:--- | :---:
`/amo/stars/indiegala-helper.svg?label=My%20Addon` |

### CODE REVIEW

File: service-tests/mozilla.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  .expectJSONTypes(Joi.object().keys({
+    name: Joi.equal('rating'),
+    value: Joi.string().regex(/^\d\/\d$/)
+  }));
+
+t.create('Stars')
+  .get('/stars/IndieGala-Helper.json')
+  .expectJSONTypes(Joi.object().keys({
+    name: Joi.equal('rating'),
+    value: Joi.string()
```
Comment (RedSparr0w) [inline service-tests/mozilla.js]: not sure how to do this one correctly, anyone got any ideas?
this is the json for 4/5 stars:
```json
{
  "name": "rating",
  "value": "â˜…â˜…â˜…â˜…â˜†"
}
```

**Comment by paulmelnikow** [comment]
Hooray, more tests! Will take a look soon!

(Assigning myself not to take ownership, but so I don't forget!)

File: service-tests/mozilla.js (hunk 1 above)
Comment (paulmelnikow) [reply service-tests/mozilla.js]: I read [the JSON spec](https://tools.ietf.org/html/rfc7159#section-7) and a few other things, and it's clear that, while escaping JSON is allowed, it's also totally acceptable to return almost any Unicode character within a JSON string, and also not necessary to specify the content type in a JSON HTTP call. Firefox and the terminal display [the JSON result](https://img.shields.io/amo/stars/IndieGala-Helper.json) just fine:

```json
{
  "name": "rating",
  "value": "★★★★☆"
}
```

I'm seeing the same as you in Chrome and Safari, and don't completely understand why.

After some playing around, it looks like `Joi.string().regex(/^[\u2605\u2606]{5}$/)` will work.

**Comment by paulmelnikow** [comment]
Looks great! Will give you a chance to update the regex and then merge. Thanks for adding more tests!

**Comment by RedSparr0w** [comment]
Thanks for that,
Have updated the regex!
Should be good to go.

**Pull request merged**